docker/e2e/Dockerfile

```diff
@@ -59,6 +59,10 @@ RUN mkdir -p /robotest
 WORKDIR /robotest
 COPY entrypoint.sh /entrypoint.sh
 COPY build/robotest-e2e /usr/bin/robotest-e2e
+COPY polaris.pub /robotest/polaris.pub
+COPY polaris.pem /robotest/polaris.pem
```

Comment:
Is the path of `polaris.pem` and `pub` right?

Reply:
Also, is this the same polaris.pem, or a new one. I think it might be better to use a different one here.